Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -28,8 +34,14 @@ struct PitchSpeller {
     )
 
     static let weightCarrying = WeightCarrying.build(from: PitchSpeller.tendencyGraph)
-    static let tendencyMask: WeightCarrying<WeightedGraph<Cross<Int,Tendency>,Double>> = weightCarrying.pullback { $0.b }
-
+    static let tendencyMask: WeightCarrying<WeightedGraph<FlowNode<Cross<Int,Tendency>>,Double>> = weightCarrying.pullback {
+        switch $0 {
+        case .source: return .down
+        case .sink: return .up
+        case .`internal`(let index): return index.b
```

Comment:
Do you have to backick this each time, or only the declaration in the enum. I think because its a proper case, it should be fine?

Reply:
Ah, okay